docs: added more info to our quickstart to it make it more helpful (#633)
ApeWorX/ape: docs/userguides/quickstart.md

sudo rm -rf **\~/.vvm**
```

## Overview

The ApeWorx project folder hierarchy:

```
project              # The root project directory
├── contracts/       # Project source files, such as '.sol' or '.vy' files
├── tests/           # Project tests, ran using the 'ape test' command
├── scripts/         # Project scripts, such as deploy scripts, ran using the 'ape run <name>' command
└── ape-config.yaml  # The ape project configuration file
```

## Enviroment Variables:

> Enviroment Variables are used to help connect you to your files or ecosytems outside of ApeWorx

Reviewer: ```suggestion
Environment Variables are used to help connect you to your files or ecosystems outside of ApeWorx.
```

Please setup environment variables (where applicable) and follow the latest instructions from the 3rd party:

> Example of a use case:

Reviewer: ```suggestion
Example use cases:
```

Reviewer: Also, maybe link to one of the plugin's READMEs

Author: are you talking about docs/userguides/plugins.md?

and how does that help?

Author: issue created and will be fixed https://github.com/ApeWorX/ape-infura/issues/28

```bash

> export WEB3_INFURA_PROJECT_ID={YOUR_INFURA_PROJECT_ID}
> 
> export WEB3_ALCHEMY_API_KEY={YOUR_ALCHEMY_KEY}

Reviewer 2: Typically I use `<something goes here>` because the arrows are not valid shell commands
```suggestion
# Used by the `ape-infura` plugin
export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>

# Used by the `ape-alchemy` plugin
export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
```

```

> Visit [infura](https://infura.io/) or [alchemy](https://www.alchemy.com/) to get your free key.

Reviewer 2: Maybe shoutout the plugin docs instead?

p.s. we can actually create very simple docs that publishes at least what the readme says under our ape docs website: https://docs.apeworx.io/ape-infura

sudo rm -rf **\~/.vvm**
```

## Overview

For more indepth information about the project please look at the [projects](docs/userguides/projects.md)
It explains the purpose of each folder and how to use them effectively.

The ApeWorx project folder hierarchy:

```
project              # The root project directory
├── contracts/       # Project source files, such as '.sol' or '.vy' files
├── tests/           # Project tests, ran using the 'ape test' command
├── scripts/         # Project scripts, such as deploy scripts, ran using the 'ape run <name>' command
└── ape-config.yaml  # The ape project configuration file
```

> 

Author: moved to projects

sudo rm -rf **\~/.vvm**
```

> If you want to deploy or test on a network. You can use any of the ape network plugins.

Reviewer: ```suggestion
If you want to deploy or test on a network, you can use any of the ape network plugins.
```

Please read and install dependencies: 
-   [ape-alchemy](https://github.com/ApeWorX/ape-alchemy)
-   [ape-hardhat](https://github.com/ApeWorX/ape-hardhat)
-   [ape-infura](https://github.com/ApeWorX/ape-infura)

> -   [ape-starknet](https://github.com/ApeWorX/ape-starknet)

Author: The goal is show we can support different type of networks to support your project. We have options. 

I want a place to index network providers. 

A bigger question is where can we index a list of plugins? I want to give a short description each of the plugins

> -   [ape-starknet](https://github.com/ApeWorX/ape-starknet)

Reviewer 2: > The goal is show we can support different type of networks to support your project. We have options.

The first 3 are ProviderAPI plugins, and the last is an EcosystemAPI plugin. So let's figure out this section a bit more.

> I want a place to index network providers.

We can do this

> A bigger question is where can we index a list of plugins? I want to give a short description each of the plugins

We can do this too, let's talk next week :wink:

## Overview

> For more indepth information about the project please look at the userguide/project.md

Reviewer: Make `userguid/project.md` a link

ape -h
```

Use `ape init` to initialize your ape project folders. Visit [userguide project](~/userguide/project.md) for more information.  

```bash
ape init

> ```

Reviewer 2: Curious if this should come at the top, logically speaking

c.doThat(sender=a)  # Make Web3 transactions
assert c.MyEvent[-1].caller == a  # Search through Web3 events
```

> ## Deployments

Reviewer: ```suggestion
## Scripts
```

And then maybe talk about how `deploy` scripts are a common type of script to make.

Author: I like that idea
